C-SharpBasic/CSharp-Assignment-2/StudentData.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace C_Sharp
+{
+    class StudentData
+    {
+
+       
+        int    Student_Id, Student_Sem;
+        string Student_Dept, Student_Name;
+        
+      
+        public override string ToString()
+        {
+            return Student_Id + "\t " + Student_Name + "\t " +
+                Student_Dept + "\t " + Student_Sem;
+        }
+
+        
+        public void Display()
+        {
+
+           
+            List<StudentData> stu = new List<StudentData>()
```

Comment:
what is stu?

Reply:
object of list StudentData. I change the name to "student".